### ISSUE TITLE
UnusedVariable false positive: string interpolation inside foreach

### ISSUE THREAD

**Issue by mpesari** [slevomat/coding-standard#566]
Here's an example. The `print` line triggers an error while the `printf` line does not.

```php
<?php

$x = 1;

foreach ([2, 4, 8] as $y) {
    print "$x => $y\n";
    // printf("%d => %d\n", $x, $y);
    $x++;
}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/0b098c563420082eaf50287a68f17d01c8abe2f3
